src/v2/view-builder/views/shared/polling.js

```diff
@@ -6,16 +6,16 @@ export default {
     this.pollingInterval = newRefreshInterval || this.options.currentViewState.refresh;
     this.countDownCounterValue = Math.ceil(this.pollingInterval / MS_PER_SEC);
     // Poll is present in remediation form
-    if (this.pollingInterval) {
+    if (this.options.currentViewState.refresh) {
```

Comment:
Passing new `newRefreshInterval` for Authenticator would trigger `this._startRemediationPolling();` instead of `this._startAuthenticatorPolling();` if it is checking for `this.pollingInterval`. Since `this.options.currentViewState.refresh` is only exist for remediation check for that instead.